control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller_test.go

```diff
@@ -1659,13 +1662,24 @@ func TestControlPlaneComponents(t *testing.T) {
 		},
 	}
 
+	pullSecret := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{Name: "pull-secret", Namespace: "hcp-namespace"},
+		Data: map[string][]byte{
+			corev1.DockerConfigJsonKey: []byte(`{}`),
+		},
+	}
```

Comment:
please move this to `componentsFakeDependencies()` function.

Reply:
sorry meant `componentsFakeObjects()` :D. You can also remove this line here now.